src/providers.ts

```diff
@@ -89,6 +89,7 @@ const providers: InternalProvider[] = [
   ["TEAMCITY", "TEAMCITY_VERSION"],
   ["TRAVIS"],
   ["VERCEL", "NOW_BUILDER"],
+  ["VERCEL", "VERCEL_ENV"],
```

Comment:
Should I add `["VERCEL"]` as well?
According to [their docs](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#system-environment-variables), `VERCEL` is a binary string (`0`/`1`) and `VERCEL_ENV` is a string representing the environment, both of which are described as:

> The [Environment](https://vercel.com/docs/concepts/projects/environment-variables#environments) that the app is deployed and running on.

I might want to add `{ "ci": false }` as well, don't I?

Reply:
Yes, `VERCEL_ENV` seems to be a runtime flag indicator and therefore is not a CI.

Reply:
@pi0 

Thank you for the comment.
I added `VERCEL` environment variable and `ci: false` flag to them.